test_agent.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/python3
```

Comment:
we dont need this file anymore, i have sync up the main branch and have included this file into the final_submission branch